woocommerce/woocommerce
# [WIP] Upgrade to pnpm 7, auto resolve peer dependencies via config

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1. On a fresh clone of WooCommerce perform a `pnpm i`. there should be no errors or warnings
2. Run a build `pnpm run build`, it should work.




### FOR PR REVIEWER ONLY:
